src/jbrl-competition/frame/index.tsx

```diff
@@ -0,0 +1,326 @@
+import React from 'react';
+import Grid from '@mui/material/Grid';
+import styled from 'styled-components';
+import orderBy from 'lodash/orderBy';
+import Button from 'common/button';
+import CenteredLoadingIndicator from 'common/centered-loading-indicator';
+import useCurrentBreakpoint from 'hooks/useCurrentBreakpoint';
+import useGqlFetchAll from 'hooks/useGqlFetchAll';
+import useDCAGraphql from 'hooks/useDCAGraphql';
+import { FullPosition, Token } from 'types';
+import { NETWORKS, POSITION_VERSION_4 } from 'config';
+import { BigNumber } from 'ethers';
+import { getDisplayToken } from 'utils/parsing';
+import useRawUsdPrices from 'hooks/useUsdRawPrices';
+import { parseUsdPrice, toToken } from 'utils/currency';
+import useUnderlyingAmount from 'hooks/useUnderlyingAmount';
+import { CREATED_AT_STOP, JBRL_CUTOFF, JBRL_ADDRESS, JBRL_TOKEN, CREATED_AT_START } from 'jbrl-competition/constants';
+import { Typography } from '@mui/material';
+import { FormattedMessage } from 'react-intl';
+import useYieldOptions from 'hooks/useYieldOptions';
+import { useHistory } from 'react-router-dom';
+import { changeMainTab } from 'state/tabs/actions';
+import { formatUnits } from '@ethersproject/units';
+import { useAppDispatch } from 'state/hooks';
+import LeaderboardSummary from '../leaderboard';
+import getPositions from '../graphql/getPositions.graphql';
+
+const StyledGrid = styled(Grid)<{ isSmall?: boolean }>`
+  ${({ isSmall }) => isSmall && 'padding-top: 28px !important;'}
+`;
+
+interface UserCompPositions {
+  user: string;
+  generatedUsd: number;
+  totalDepositedUsd: number;
+  totalUsedUsd: number;
+  totalUsed: BigNumber;
+  totalDeposited: BigNumber;
+  generated: { token: Token; amount: BigNumber }[];
+  positions: FullPosition[];
+}
+
+const HomeFrame = () => {
+  const currentBreakPoint = useCurrentBreakpoint();
+  const history = useHistory();
+  const dispatch = useAppDispatch();
+  const client = useDCAGraphql(Number(NETWORKS.polygon.chainId), POSITION_VERSION_4);
+  const {
+    loading: isLoadingPositions,
+    data,
+    // refetch,
+  } = useGqlFetchAll<{ positions: FullPosition[] }>(
+    client,
+    getPositions,
+    {
+      createdAtStart: CREATED_AT_START.toString(),
+      createdAtStop: CREATED_AT_STOP.toString(),
+    },
+    'positions'
+  );
+
+  const onCreatePosition = () => {
+    dispatch(changeMainTab(0));
+    history.push('/create/137/0xf2f77fe7b8e66571e0fca7104c4d670bf1c8d722/');
+  };
+
+  const [yieldOptions, isLoadingYieldOptions] = useYieldOptions(NETWORKS.polygon.chainId);
+
+  const positions = React.useMemo(() => data?.positions || [], [data]);
```

Comment:
Why are you memoizing a field reference?

Reply:
So it doesn't change on every re-render due to reference checking